styles.css

```diff
@@ -0,0 +1,712 @@
+    backdrop-filter: blur(0);
+    padding-top: 20px;
+    padding-right: 0;
+    background-color: transparent;
+    text-align: center;
+    transition: 0.3s;
+    width: 70%;
+    gap: 35px;
+    color: #344563;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 27px;
+    line-height: 44px;
+    display: flex;
+  }
+
+  .nav-link {
+    display: flex;
+    font-style: normal;
+    font-weight: 900;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    transition: 0.7s ease;
+  }
+
+  #nav1 {
+    color: #ec5242;
+  }
+
+  .nav-item {
+    padding-left: 0;
+    margin-right: 0;
+  }
+
+  .upper-nav {
+    display: block;
+    background-color: #383839;
+  }
+
+  .top {
+    display: flex;
+    gap: 17px;
+    padding: 5px;
+    margin-left: 56%;
+  }
+
+  .pa {
+    color: white;
+    font-size: 12px;
+  }
+
+  #fb {
+    width: 16px;
+    height: 16px;
+  }
+
+  #twitter {
+    width: 16px;
+    height: 16px;
+  }
+
+  .GE {
+    padding-left: 0;
+    height: 25px;
+    color: #ec5242;
+    border: 3px solid #ec5242;
+  }
+
+  #ge {
+    color: #ec5242;
+    font-weight: 900;
+    padding: 0 15px;
+  }
+
+  #logo {
+    width: 40px;
+    margin-left: 120px;
+    margin-top: 11px;
+    height: 40px;
+  }
+
+  .flex {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .flex .logo {
+    display: flex;
+    width: 61px;
+  }
+
+  header {
+    width: 51%;
+  }
+
+  .grid-item2 {
+    display: none;
+  }
+
+  .grid-section {
+    grid-template-columns: auto auto auto auto auto;
+    column-gap: 5px;
+    margin-top: 5%;
+  }
+
+  .item1 {
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
+    align-items: center;
+  }
+
+  .main-program {
+    padding-bottom: 8%;
+    background: url(assets/pat.jpg);
+    background: #383839;
+  }
+
+  .heading31 {
+    padding-top: 15px;
+    color: #ffff;
+    display: flex;
+    justify-content: center;
+    margin-top: 5%;
+  }
+
+  .hrprog {
+    display: block;
+    width: 20%;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 10px;
+  }
+
+  .more-professors-btn {
+    display: none;
+  }
+
+  .professor-cards {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    padding: 0 10%;
+  }
+
+  .partner-icons-list {
+    flex-direction: row;
+    gap: 47px;
+  }
+
+  .footer {
+    padding: 15px 200px;
+  }
+
+  #footer-about {
+    background-color: #272a31;
+    height: 220px;
+    justify-content: center;
+  }
+
+  .partners {
+    display: none;
+  }
+
+  #br {
+    display: none;
+  }
+
+  .group-photo {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    padding: 35px 120px;
+  }
+
+  .grp {
+    width: 94%;
+    height: 90%;
+    padding: 0;
+  }
+
+  #earth {
+    width: 100px;
+    height: 100px;
+  }
+
+  .copyright {
+    margin-top: 37px;
+    color: #fff;
+  }
+
+  .heading1about {
+    display: none;
+  }
+
+  .heading2 {
+    font-size: 70px;
+  }
```

Comment:
- [ ] If you notice from the original design, you can see that the heading has the image inside the text. Kindly consider applying that to ensure that your design matches the original design ☺️.

Your design 😎 
![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/99394374/9a44b722-e5e4-4df3-bdf4-6a6511dcb957)

Original design 😎 
![image](https://github.com/fatmahussein/Green-Earth-Summit/assets/99394374/40affcdf-a8e2-41fb-830b-afad873b27a4)